### PR TITLE
Create celsojr.json

### PR DESCRIPTION
**Pull request by celsojr**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://celsojr.com

### CODE REVIEW

**Comment by Satindar31** [comment]
This is probably going to be declined because your website has very little content. Add more stuff like maybe an about me for it to be accepted.

**Comment by celsojr** [comment]
I have added some content before migrating the full website over here: https://celsojr.com

**Comment by Satindar31** [comment]
Looks good now!

File: domains/celsojr.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "CelsoJr's developer personal website",
+  "repo": "https://github.com/celsojr/celsojr",
+  "owner": {
+    "username": "celsojr",
+    "email": "celsojrfull@gmail.com"
+  },
+  "record": {
+    "CNAME": "celsojr.github.io/celsojr"
```
Comment (CuteDog5695) [inline domains/celsojr.json:9]:
```suggestion
    "CNAME": "celsojr.github.io"
```

**Review by iostpa — DISMISSED** [review]

**Pull request merged**